src/Paynow/Exception/PaynowException.php

```diff
@@ -18,13 +18,12 @@ public function __construct($message = '', $code = 0, $body = null)
                 foreach ($json->errors as $error) {
                     $this->errors[] = new Error($error->errorType, $error->message);
                 }
-                $this->errors = $json->errors;
             }
         }
     }
 
     /**
-     * @return mixed
+     * @return array|null
```

Comment:
you can improve this annotation by adding a type of elements in the collection since it's a part of public API

Reply:
Done